lib/Invoice.php

```diff
@@ -58,9 +58,9 @@
  * @property int $subscription_proration_date
  * @property int $subtotal
  * @property int $tax
- * @property float $tax_percent
  * @property mixed $threshold_reason
```

Comment:
you're missing `default_tax_rates` too: https://stripe.com/docs/api/invoices/object#invoice_object-default_tax_rates

Reply:
Damn. This is not the golden standard I've grown to expect from myself. Fixed.